### PR TITLE
Isochrone: fix a bug

### PR DESCRIPTION
**Pull request by ZoeBrunet**
Correction in project_in_direction to avoid boost:geometry exception in
merge poly.

### CODE REVIEW

File: source/routing/isochrone.cpp [hunk 1]
```diff
@@ -118,7 +118,7 @@ type::Polygon circle(const type::GeographicalCoord& center,
     return points;
 }
 
-type::MultiPolygon merge_poly(const type::MultiPolygon& multi_poly, type::Polygon poly) {
+static type::MultiPolygon merge_poly(const type::MultiPolygon& multi_poly, type::Polygon poly) {
```
Comment (pbougue) [inline source/routing/isochrone.cpp:121]: :+1:

**Comment by pbougue** [comment]
Ok for me

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by pbougue** [comment]
Jenkins, test this please.

**Pull request merged**